include/vctr/Expressions/DSP/Decibels.h

```diff
@@ -82,7 +82,7 @@ struct dBPower : Constant<10> {};
    @ingroup Expressions
  */
 template <is::constant DecibelConstant, auto minDb = -100>
-constexpr inline ExpressionChainBuilder<expressions::MagToDb, DecibelConstant, Constant<minDb>> magToDb;
+constexpr inline ExpressionChainBuilderWithRuntimeArgs<expressions::MagToDb, detail::RuntimeArgChain<std::tuple<>, std::tuple<>, std::tuple<>>, DecibelConstant, Constant<minDb>> magToDb;
```

Comment:
This is the quick fix: If the expression does not need any runtime args, `ExpressionChainBuilder` just creates a dummy `RuntimeArgChain` with a single empty tuple. Since these expressions are compositions actually made up of 3 expressions, manually specifying a `RuntimeArgChain` with three empty tuples fixes it. In the long term, `ExpressionChainBuilder` should be able to figure out if the expression type passed in is a composition of multiple expressions and do that automatically, since this syntax is super confusing

Reply:
does this break anything? looks like an interface change but cannot really tell from the context..

Reply:
No this does not break anything. The object relevant to the user is still named `vctr::magToDb` and works correct now if chained with expressions that need the `applyRuntimeArgs` call to work correctly.